nss-day-cohort-46/sprinkles-of-joy-kylo-ren-s-rice-krispie-treats
# feat: orderHistory, when you click orderHistory on custNav, orders ar…

# Description
Given the user is on the nav bar
When the user clicks on an order hyperlink
Then the user will be presented with the order detail including all the products for that order with order status, total, and timestamp

Fixes # (issue)
feat: orderHistory

## Type of change
Please delete options that are not relevant.


# Testing Instructions

`git fetch --all`
`git checkout nah-orderHistory-viewOrder`
SERVE

1. Once you pull down the branch.. you will need to login as one of our users.
2. You can add items to the cart and placeOrder... if you want to see your order added to the history.
3. Once logged in, you click OrderHistory on custNav and you will see the orders render to the page.

# Checklist:
